ml_models/use_model.py

```diff
@@ -63,10 +69,29 @@ def get_answer(query):
     ans_list = [get_text_by_ind(ind) for k, ind in indexer.return_closest(query, k=4)]
     return ans_list
 
+
 def get_answer_ind(query):
     ind_list = [get_new_ind_by_ind(ind) for k, ind in indexer.return_closest(query, k=4)]
     return ind_list
 
+
+def get_slack_answer_ind(query, k=4):
+    import requests
+
+    url = f"https://slack.com/api/search.messages?count={k}&query={query}&pretty=1"
+
+    payload = {}
+    headers = {
+        'Authorization': f'Bearer {os.environ["USER_TOKEN"]}'
+    }
+
+    response = requests.request("GET", url, headers=headers, data=payload)
+    matches = json.loads(response.text).get('messages').get('matches')
+    ind_list = ['_'.join((match.get('user'), re.sub(r'\.\d+', r'.0', match.get('ts')))) for match in matches] #TODO in labelled_all.csv at the end of the timestamp always zero
```

Comment:
You should extract a function with the processing of match obj. In code, it seems complicated.

Reply:
Further, you can add tests on this function with examples.

Reply:
done